README.md

```diff
@@ -1,2 +1,39 @@
-# Awesome books-with ES6
- 
+# Awesome books: with ES6
+
+> this my Awesome books version
+
+## Built With
+
+- HTML
+- CSS
+- javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://x-elie-x.github.io/Awesome-books-with-ES6/index.html)
+
+## Getting Started
```

Comment:
- [ ] Kindly add the get started section to your Readme file. give more detail on how someone can clone your project and get started.